Yafc.UI/Core/WindowUtility.cs

```diff
@@ -59,18 +62,12 @@ protected override void MainRender() {
     }
 
     protected internal override void Close() {
+        if (parent != null && !parent.closed)
+            parent.removeChildWindow(this);
```

Comment:
The style checker is probably going to balk at the lack of curly braces here
```suggestion
        if (parent != null && !parent.closed) {
            parent.removeChildWindow(this);
        }
```